.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+    ignore = E203, E266, E501, W503, F403, F401
+    max-line-length = 79
```

Comment:
In de DSO config zie ik een waarde van 99?

Reply:
Ja aangepast, zodat het conform de DSO settings is